packages/ui/src/views/repo/repo-summary/components/summary.tsx

```diff
@@ -0,0 +1,135 @@
+import { useNavigate } from 'react-router-dom'
+
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+  ButtonGroup,
+  Icon,
+  Spacer,
+  StackedList,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text
+} from '@/components'
+import { CommitCopyActions } from '@/components/commit-copy-actions'
+import { getInitials } from '@/utils/utils'
+
+import { FileProps, SummaryItemType } from '../repo-summary.types'
+
+interface SummaryProps {
+  latestFile: Pick<FileProps, 'user' | 'lastCommitMessage' | 'timestamp' | 'sha'>
+  files?: FileProps[]
```

Comment:
Again, these aren't `Props`, can we reserve the `Props` suffix for types that describe the props of a component. In this case, maybe `FileItem` or `RepoFile`?